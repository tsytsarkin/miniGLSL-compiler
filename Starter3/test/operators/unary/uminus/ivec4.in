{ ivec4 x; y = -x; }
