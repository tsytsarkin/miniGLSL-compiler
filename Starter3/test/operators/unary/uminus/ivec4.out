  1: { ivec4 x; y = -x; }
(SCOPE (DECLARATIONS (DECLARATION x ivec4 ) ) (STATEMENTS (ASSIGN unknown y (UNARY ivec4 - x ) ) ) ) 
Failed to compile
